.github/workflows/pr-comment-terraspace.yaml

```diff
@@ -0,0 +1,189 @@
+name: Terraspace Comment
+
+on:
+  issue_comment:
+    types:
+      - created
+
+env:
+  AWS_REGION: us-east-1 # TODO this should be dynamic
+  TF_IN_AUTOMATION: 1
+
+jobs:
+  pre-reqs:
+    runs-on: ubuntu-latest
+    outputs:
+      result: ${{ steps.detector.outputs.result }}
+      gitRef: ${{ steps.git_ref.outputs.result }}
+    steps:
+      - name: Determine Git Ref
+        uses: actions/github-script@v6
+        id: git_ref
+        with:
+          result-encoding: string
+          script: |
+            if (context.eventName === "pull_request") {
```

Comment:
Does it ever go inside that IF even though you configured this workflow to run only on issue comments?

Reply:
good spot, this isn't needed anymore